### DISCUSSION TITLE
I wanted to test a transaction automation. Now, my account is suspended. The shop was in developer mode.

### DISCUSSION THREAD

**Discussion by KevinMichaelSchindler** [etsy/open-api#1305] [Show and tell]
I am honestly speechless.

You can’t test properly because no separate environment is provided. I made one purchase, and my account got suspended. After appealing, it was rejected, of course, without disclosing the reason.

I wanted to set up automation for a client and test it on my end, and this is what I get.

I’m at a loss for words. 

Is this happening often?

**Comment by Wilderbill** [comment]
I think they are being flooded with sellers, buyers and coders and
are perfectly happy to turn some away for any reason, in their humble opinion.
It may even be risky for me to make this observation.

**Reply by KevinMichaelSchindler** [reply]
Ye, I guess so. Well, I moved on and I am 10x happier. This Etsy dumpster fire is just annoying. Sellers complain, buyers complain and well now I guess devs complain. Anyways have nothing good to say about this experience but happy that I am not involved anymore.

**Reply by Gareth064** [reply]
99% of all sales for my wifes business is form Etsy. We have our own Shopify store, but getting traffic to it is costly. If sales didn't come as easy on Etsy then I'd be saying good bye as well. I have been trying to build a "Quality of life" type of app over Etsy to do things for my wife's business that you can't do well or can't do at all on Etsy. But since they used us to beta test the hell out of their new API, the API devs have pretty much abandoned the developer community.
